pkgs/by-name/eb/eb-garamond/package.nix

```diff
@@ -1,23 +1,31 @@
-{
-  lib,
-  stdenvNoCC,
-  fetchzip,
-}:
-
+{ lib, pkgs, stdenvNoCC, fetchFromGitHub }:
 stdenvNoCC.mkDerivation rec {
   pname = "eb-garamond";
   version = "0.016";
 
-  src = fetchzip {
-    url = "https://bitbucket.org/georgd/eb-garamond/downloads/EBGaramond-${version}.zip";
-    hash = "sha256-P2VCLcqcMBBoTDJyRLP9vlHI+jE0EqPjPziN2MJbgEg=";
+  src = fetchFromGitHub {
+    owner = "georgd";
+    repo = "EB-Garamond";
+    rev = "d04d0f9da";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
I'm aware this causes issues with the build script, but we usually don't fetch by commit hash if a tag is available. Can you pass a dummy value (or even just version string) as commit rev to the makefile instead?